assets/blocks/quiz/question-block/question-block.editor.scss

```diff
@@ -102,15 +102,17 @@ $block: '.sensei-lms-question-block';
 				font-size: 12px;
 				height: auto;
 				text-transform: uppercase;
-				background: #111;
+				background: #fff;
+				border: 1px solid currentColor;
+				color: currentColor;
```

Comment:
Need to set this to a fixed black color (`#1e1e1e` in the editor), because `currentColor` is the content text color, which can be white/light too.

Reply:
👌  https://github.com/Automattic/sensei/pull/4031/commits/39d1237c62580f92933eb67956875a35bf6feccc